BaseFilter/sections/specific.txt

```diff
@@ -2583,6 +2583,11 @@ sbfull.com##div[style^="position:fixed;inset:0px;z-index:"]
 canuckaudiomart.com##div[id^="replacement-position-"]
 canuckaudiomart.com##div[style^="min-width:300px;min-height:250px;"]
 insiderintelligence.com##div[class^="TopAd__"]
+insiderintelligence.com##div[class^="topic__AdTopWrapper-"] > div[class^="sc-"]
+insiderintelligence.com##div[class^="topic__AdBottomWrapper-"]
+insiderintelligence.com##div[class*="Content__RightRailAdWrapper-"]
+insiderintelligence.com###div-id-for-middle1-ad
+||insiderintelligence.com/scripts/analytics/ads.js
```

Comment:
Please look at a content. It should not be blocked.

Reply:
Excuse me.
I have checked with the Dev tool. 
But I think they are placeholder from google_ads_iframe.
I also think that `insiderintelligence.com/scripts/analytics/ads.js` is also a script that generates the frame for the ad.
Which rule is bad? Is it all of rules?
If you don't mind, could you please let me know?

Reply:
I see. Thank you for teach me.
I will remove that rule.